api/src/main/java/org/hawkular/apm/api/utils/PropertyUtil.java

```diff
@@ -191,12 +194,14 @@ public static Integer getPropertyAsInteger(String name, Integer def) {
      *
      * @return The property as a boolean, or false if not found
      */
-    public static boolean getPropertyAsBoolean(String name) {
+    public static Boolean getPropertyAsBoolean(String name) {
```

Comment:
I have reverted this change, Before I tried to do some "magic" with getting typed properties, I forgot that I have changed the method signature.